frontend/test/metabase/scenarios/collections/bookmarks.cy.spec.js

```diff
@@ -0,0 +1,92 @@
+import _ from "underscore";
+import { restore, sidebar } from "__support__/e2e/cypress";
+
+describe("Bookmarks in a collection page", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("cannot add bookmark to root collection", () => {
+    cy.visit("/collection/root");
+
+    cy.icon("bookmark").should("not.exist");
+  });
+
+  it("can add and remove bookmarks from collection from its page", () => {
+    cy.visit("/collection/1");
+
+    // Add bookmark
+    cy.icon("bookmark").click();
+
+    // Bookmark Icon should be blue once Collection is bookmarked
+    cy.icon("bookmark")
+      .should("have.css", "color")
+      .and("eq", "rgb(80, 158, 227)");
```

Comment:
I think it'd make sense to cover this with unit tests using `color("brand")` instead of a hardcoded RGB value. This can make the test fragile